src/main/java/com/example/quizapp/quiz/QuizThumbnail.java

```diff
@@ -37,14 +41,32 @@ public QuizThumbnail(Quiz quiz) {
         }
         this.quiz = quiz;
         quizName.setText(quiz.getName());
+        if (isMyQuiz()) {
+            fxmlLoader = new FXMLLoader(getClass().getResource("/fxml/editButton.fxml"));
+            EditButton editButton = new EditButton(quiz);
+            fxmlLoader.setController(editButton);
+            fxmlLoader.setRoot(editButton);
+            try {
+                AnchorPane editButton1 = fxmlLoader.load();
```

Comment:
Kanske tydligare namn på editButton och editButton1, är editButton1 där vi laddar in edit knappen?